crates/ruff_linter/src/rules/pydocstyle/rules/indent.rs

```diff
@@ -291,10 +291,21 @@ pub(crate) fn indent(checker: &mut Checker, docstring: &Docstring) {
             let line_indent = leading_space(last);
             let line_indent_size = line_indent.chars().count();
             if line_indent_size > docstring_indent_size {
+                let mut offset_len = line_indent.text_len();
+                // When a non-space character is on the last line, the trailing quotes are also on the same line.
+                // No diagnostic is added if the indentation difference equals the indent size.
+                if last.chars().any(|c| !c.is_whitespace()) {
+                    let prev_line_indent = leading_space(&lines[lines.len() - 2]);
```

Comment:
I think this might panic if there are fewer than 3 lines because it then results in an out of bound access.

We should also consider the case where there's a number of blank lines between the last and previous line


```python
def right_indent_quotes_same_line(a: int) -> None:
    """Foo.
    Parameters
    ----------
    a : int





        A parameter."""
    pass
```